openstack_controller/datadog_checks/openstack_controller/check.py

```diff
@@ -471,3 +612,218 @@ def _report_load_balancer_response_time(self, api, project_id, project_tags):
             self.service_check('openstack.octavia.api.up', AgentCheck.OK, tags=project_tags)
         else:
             self.service_check('openstack.octavia.api.up', AgentCheck.UNKNOWN, tags=project_tags)
+
+    def _report_load_balancer_loadbalancers(self, api, project_id, project_tags):
+        loadbalancers_data = api.get_load_balancer_loadbalancers(project_id)
+        if loadbalancers_data is not None:
+            for loadbalancer_id, loadbalancer_data in loadbalancers_data.items():
+                loadbalancer_tags = _create_load_balancer_loadbalancer_tags(loadbalancer_data)
+                all_tags = loadbalancer_tags + project_tags  # TODO: add loadbalancer api tags
+
+                # report status
+                self.gauge(
```

Comment:
nit- could we use `get_normalized_metrics` for these metrics and make a list of load balance metrics to collect?